src/script/cpp_api/s_node.cpp

```diff
@@ -133,7 +133,8 @@ bool ScriptApiNode::node_on_dig(v3s16 p, MapNode node,
 	push_v3s16(L, p);
 	pushnode(L, node, ndef);
 	objectrefGetOrCreate(L, digger);
-	PCALL_RES(lua_pcall(L, 3, 0, error_handler));
+	pushPointedThing(pointed);
+	PCALL_RES(lua_pcall(L, 4, 0, error_handler));
```

Comment:
`on_dig` in nodedef will also have a new parameter. Please add documentation.

Reply:
true.